docs/source/notes/get_start_xpu.rst

```diff
@@ -1,109 +1,81 @@
-Pytorch 2.4: Getting Started on Intel GPU
-=========================================
+Getting Started on Intel GPU
+============================
 
-The support for Intel GPUs is released alongside PyTorch v2.4.
-
-This release only supports build from source for Intel GPUs.
-
-Hardware Prerequisites
-----------------------
+Hardware Prerequisite
+---------------------
 
 .. list-table::
+   :widths: 50 50
    :header-rows: 1
 
-   * - Supported Hardware
-     - Intel® Data Center GPU Max Series
-   * - Supported OS
+   * - Validated Hardware
+     - Supported OS
+   * - Intel® Data Center GPU Max Series
      - Linux
+   * - Intel Client GPU
+     - Windows/Linux
+
+Intel GPUs support enhancement is ready in PyTorch* 2.5 for Intel® Data Center GPU Max Series and Intel® Client GPUs on both Linux and Windows, which brings Intel GPUs and the SYCL* software stack into the official PyTorch stack with consistent user experience to embrace more AI application scenarios.
```

Comment:
Would it be good to call out that this support is still in an experimental / beta phase considering 2.5 is the first release with actual binaries?

Also considering we don't have a consistent CI with these builds I'd be worried that we're giving users the wrong impression that these are very well battle tested

Reply:
Sure, let me add it.

Reply:
Done

Reply:
Hi @seemethere , here is the list of CI for Intel GPU 
https://github.com/pytorch/pytorch/blob/main/.github/workflows/xpu.yml